apps/users/serializers/user.py

```diff
@@ -205,6 +209,7 @@ class Meta:
             "can_public_key_auth",
             "mfa_enabled",
             "need_update_password",
+            "is_face_code_set",
         ]
         # 包含不太常用的字段，可以没有
         fields_verbose = (
```

Comment:
The code is fine and does not include any abnormalities, potential issues, or unnecessary complexities. It follows best practices for Django's ORM models. If you need help with specific aspects of the code, please provide further clarification.